src/algebra/continued_fractions/computation/basic.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2020 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import algebra.continued_fractions.basic
+import algebra.ordered_field
+import algebra.archimedean
+/-!
+# Computable Continued Fractions
+
+## Summary
+
+We formalise the standard computation of regular continued fractions for linear ordered floor
+fields. The algorithm is rather simple and connected to the Euclidean algorithm. It can be found,
+for example, on Wikipedia:
+https://en.wikipedia.org/wiki/Continued_fraction#Calculating_continued_fraction_representations.
+
+## Main definitions
+
+- `generalized_continued_fraction.of`: computes the generalised continued fraction of a value.
```

Comment:
In the file, you have `stream` and `seq1`, which also belong to the "Main definitions". You should mention them here, explain how they are constructed. In an "Implementation" section, you should also probably explain why you need the two of them.

Reply:
Done